dtfs-central-api/src/v1/validation/utilisation-report-service/utilisation-report-validator.js

```diff
@@ -35,7 +35,7 @@ const validateReportId = (reportId) => {
  * @returns {string | null} - Error message or null if valid.
  */
 const validateMonth = (month, propertyName = 'Month') => {
-  if (!month) {
+  if (!month && month !== 0) {
```

Comment:
mmm javascripty falsey values

Reply:
Yep - something that would technically work fine, but our error messages seem specific so I wanted our check to be specific too (caught this in one of the unit tests I added)